Dockerfile

```diff
@@ -9,9 +9,11 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 RUN \
   apt-get update && \
   apt-get install -y --no-install-recommends --no-install-suggests wget curl software-properties-common docker.io && \
+  rm -rf /var/lib/apt/lists/* && \
```

Comment:
Isn't the second on you added enough?

Reply:
Must be after every apt-get install, but we can remove the last install and rm to make one line with apt install